### DISCUSSION TITLE
how can i delete the specfic graphics in pdf

### DISCUSSION THREAD

**Discussion by marc0511** [pymupdf/PyMuPDF#1432] [Q&A]
hi, there

i have some pdf document, and each page have the same graphics(logo) in there, is there possible delete them by pymupdf . thank you.

**Comment by JorjMcKie** [comment]
It depends on whether the logo is an image or a drawing (it is not the same thing).
An image would be no problem - just use redaction annotations.
A drawing might be possible but could be hacky - depends on the concrete case.

**Comment by marc0511** [comment]
Thanks response me!

i am very sure the graphics is drawing.  pretty difficult case for me haha . i was tried locate the graphic with xref, and use deletet object to remove the drawing , but it cant work.

**Comment by JorjMcKie** [comment]
Let me have a look at the PDF if you like.
